//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// acefg_T_A.h
//
// Code generation for function 'acefg_T_A'
//

#ifndef ACEFG_T_A_H
#define ACEFG_T_A_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void acefg_T_A(double V_init, double A_init, double A_wayp, double V_max,
               double A_min, double J_max, double J_min, double T,
               creal_T t[14]);

#endif
// End of code generation (acefg_T_A.h)
